libsolidity/codegen/YulUtilFunctions.cpp

```diff
@@ -1652,22 +1653,24 @@ string YulUtilFunctions::copyByteArrayToStorageFunction(ArrayType const& _fromTy
 				switch gt(newLen, 31)
 				case 1 {
 					let loopEnd := and(newLen, not(0x1f))
+					<?fromStorage> src := <srcDataLocation>(src) </fromStorage>
 					let dstPtr := dstDataArea
 					let i := 0
 					for { } lt(i, loopEnd) { i := add(i, 32) } {
```

Comment:
Maybe, to be consistent with the rest of this function?
```suggestion
					for { } lt(i, loopEnd) { i := add(i, 0x20) } {
```